packages/db/src/types.ts

```
@@ -1,108 +1,103 @@
+) => Promise<DbGetResult>;
 
 type GetMinMax2<T> = (
   table: T,
   targetField: string,
   fields: string[],
   filterFields1: Record<string, string | number | Array<string | number>>,
   filterFields2: Record<string, string | number | Array<string | number>>,
-  order?: string
-) => Promise<DbGetResult>
+  order?: string,
+) => Promise<DbGetResult>;
```

Biome (2.4.9)

[error] Function has 6 parameters, but only 5 are allowed.

(lint/complexity/useMaxParams)